### DISCUSSION TITLE
Handle external links

### DISCUSSION THREAD

**Discussion by froaderick** [dfinke/ImportExcel#1464] [Ideas]
It would be useful (to me, anyway) to have an parameter in Get-ExcelSheetInfo, or maybe a new cmdlet, that identifies if the sheet has a cell that contains an external link.  We do lots of file share to Sharepoint migrations and that would be useful.

Unless, of course, there is already something in this module that does that and i missed it? (always possible i missed something)

Thanks!

**Comment by dfinke** [comment]
Thanks for the post. Sounds interesting. Could you say a bit more? What is your definition of an external link?
This operation would be expensive when the number of rows and columns are large. Each cell in the sheet would need to be inspected.
